.gitignore

```diff
@@ -8,6 +8,14 @@
 # Test binary, built with `go test -c`
 *.test
 
+# Ignore the server binary output by `go build`
+server
```

Comment:
server is too generic, If in future if someone wants to add server folder it would be hard for them to find why the server folder is not reflected in the PR. Remove it or find a way that only server binary is removed

Reply:
![image](https://github.com/keploy/keploy/assets/114267538/b8ca2d87-dd93-421a-a13a-faa6cf780679)
 
 With this logic now, any folders named server will be recognized by git and only server binary will be ignored by git.